terraform/meta/main.tf

```diff
@@ -43,4 +43,7 @@ module "prod_backend" {
 
 module "ecr" {
   source = "../modules/ecr"
+
+  # This will need updating to include dev and production roles

```

Comment:
We'd need to create the roles in dev and prod, and I don't think we want to be applying anything there yet.

Reply:
I think we might forget to add these in once we've created the roles in and dev and prod, however it will probably be quickly apparent what's missing so don't think it's worth creating a ticket / updating this comment to a TODO for